frontend/src/images/openvino-2.svg

```diff
@@ -0,0 +1 @@
3018)"/><path d="m190.772 192.206c.555 3.333 1.875 5.972 3.958 7.916 2.083 1.945 4.681 2.917 7.792 2.917 2.277 0 4.458-.653 6.541-1.958 2.084-1.306 3.848-3.125 5.292-5.459l7.333 4.584c-2.444 3.5-5.194 6.069-8.25 7.708-3.055 1.639-6.694 2.458-10.916 2.458-6.278 0-11.389-1.944-15.334-5.833-3.944-3.889-5.916-9-5.916-15.333 0-6.334 1.986-11.542 5.958-15.625 3.972-4.084 9.014-6.125 15.125-6.125 6.056 0 10.986 2.014 14.792 6.041 3.805 4.028 5.708 9.237 5.708 15.625l-.167 3.084zm0-6.75h22.333c0-3.278-1.014-5.903-3.042-7.875-2.027-1.972-4.625-2.959-7.791-2.959-3 0-5.514.945-7.542 2.834s-3.347 4.555-3.958 8z" transform="matrix(1.11749 0 0 1.11749 -72.67088 -164.53018)"/><path d="m229.438 168.539h9.25v6.25c3.056-5 7.389-7.5 13-7.5 2.056 0 4 .389 5.834 1.167 1.833.778 3.375 1.861 4.625 3.25s2.097 2.903 2.541 4.541c.445 1.639.667 4.014.667 7.125v25.667h-9.25v-23.333c0-1.778-.111-3.375-.333-4.792-.223-1.417-1.07-2.694-2.542-3.833s-3.181-1.709-5.125-1.709c-2.944 0-5.25 1.028-6.917 3.084-1.666 2.055-2.5 4.889-2.5 8.5v22.083h-9.25z" transform="matrix(1.11749 0 0 1.11749 -72.49618 -164.34418)"/><path d="m277.605 148.956 13.917 40.083c.777 2.445 1.333 5.472 1.666 9.083.334-3.944.917-6.972 1.75-9.083l13.834-40.083h10.416l-22.083 60.083h-7.75l-22.083-60.083z" transform="matrix(1.11749 0 0 1.11749 -74.63808 -164.34418)"/><path d="m285.70459478 2.11266044h10.80277583v67.14215167h-10.80277583z" stroke-width="1.11749"/><path d="m339.272 148.956h8.75l32.25 40.333c1.333 1.667 2.722 4.278 4.166 7.833-.389-1.944-.722-3.819-1-5.625-.277-1.805-.416-3.013-.416-3.625v-38.916h9.666v60.083h-8.5l-32.666-40.667c-1.389-1.666-2.75-4.138-4.084-7.416l.417 1.916.333 2.084c.5 2.778.75 4.361.75 4.75v39.333h-9.666z" transform="matrix(1.04993 0 0 1.11749 -51.11798 -164.34418)"/><path d="m430.855 147.706c8.944 0 16.417 3 22.417 9s9 13.444 9 22.333-3 16.333-9 22.333-13.473 9-22.417 9-16.403-2.986-22.375-8.958-8.958-13.43-8.958-22.375c0-8.889 2.972-16.333 8.916-22.333 5.945-6 13.417-9 22.417-9zm-21.75 31.333c0 6.222 2.111 11.5 6.333 15.833 4.223 4.334 9.361 6.5 15.417 6.5 6.111 0 11.278-2.138 15.5-6.416s6.333-9.584 6.333-15.917c0-6.222-2.111-11.5-6.333-15.833-4.222-4.334-9.389-6.5-15.5-6.5s-11.264 2.153-15.458 6.458c-4.195 4.306-6.292 9.597-6.292 15.875z" transform="matrix(1.11749 0 0 1.11749 -80.03748 -164.53018)"/></g><path d="m67.795 88.342c.599 1.614.926 3.36.926 5.181 0 8.232-6.683 14.914-14.914 14.914s-14.914-6.682-14.914-14.914c0-8.231 6.683-14.914 14.914-14.914 1.821 0 3.567.327 5.181.926-1.854 1.125-3.094 3.163-3.094 5.488 0 3.54 2.874 6.413 6.414 6.413 2.325 0 4.362-1.239 5.487-3.094z" fill="#6c04f4" transform="translate(347.91052 -57.39018)"/><path d="m67.795 88.342c.599 1.614.926 3.36.926 5.181 0 8.232-6.683 14.914-14.914 14.914s-14.914-6.682-14.914-14.914c0-8.231 6.683-14.914 14.914-14.914 1.821 0 3.567.327 5.181.926-1.854 1.125-3.094 3.163-3.094 5.488 0 3.54 2.874 6.413 6.414 6.413 2.325 0 4.362-1.239 5.487-3.094z" fill="#6c04f4" transform="translate(-19.16058 -57.39018)"/></svg>
```

Comment:
File name should be `openvino.svg` to match what is in the application definition file.

Reply:
fixed